src/deterministic_vi/elbo_kl.jl

```diff
@@ -83,48 +83,61 @@ end
 # Subtracting KL divergences from sources #
 ###########################################
 
-subtract_kl_a(vs) = -(categorical_kl(vs[ids.a[:, 1]], prior.a))
+kl_source_a(vs) = categorical_kl(vs[ids.a[:, 1]], prior.a)
 
-function subtract_kl_r(vs)
+function kl_source_r(vs)
```

Comment:
The functions were all subtraction original so that we wouldn't have to allocate memory. But that's not an issue with ReverseDiff, so we may as well compute interpretable quantity, a term of the kl divergence, and then subtract it later